tests/src/test/java/org/eclipse/hono/tests/amqp/AmqpConnectionIT.java

```diff
@@ -262,6 +262,7 @@ public void testConnectFailsForNonMatchingTrustAnchor(final TestContext ctx) thr
      * @param ctx The vert.x test context.
      */
     @Test
+    @Ignore("the tenant service should disallow the payload")
```

Comment:
remove this test and instead add an integration test to the `org.eclipse.hono.tests.registry.Tenant*IT` classes